pages/linux/acpi.md

```diff
@@ -21,3 +21,7 @@
 - Show all information:
 
 `acpi -V`
+
+- Use /proc instead of /sys:
```

Comment:
I'm not sure about this description:
```suggestion
- Extract information from proc instead of new sys interface:
```

Reply:
Hrm. What about this, @navarroaxel?
```suggestion
- Extract information from `/proc` instead of `/sys`:
```

Reply:
the command help says:
```
-p, --proc               use old proc interface instead of new sys interface
```
but it's ok if `/proc` is well known by the community.

Reply:
Yeah, I'd prefer to mention the file page here @navarroaxel. That text from the man page doesn't real particularly well, so I think it's just easier to mention the file paths directly (as they are indeed well-known)

Reply:
Please @Ray6464, could you apply the suggestion from @sbrl to merge this?